var/spack/repos/builtin/packages/symlinks/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Symlinks(MakefilePackage):
+    """Scan or change symbolic links."""
+
+    homepage = "http://ibiblio.org/pub/Linux/utils/file"
+    url      = "http://ibiblio.org/pub/Linux/utils/file/symlinks-1.4.tar.gz"
+
+    version('1.4', sha256='4818a3be253b53f2547fe51349652c87301794fc2ff4e3104850301ffe9843a0')
+
+    def edit(self, spec, prefix):
+        filter_file(r'(/usr/local)', prefix, 'Makefile')
```

Comment:
I don't think the parentheses are necessary here are they? It might be better to use `string=True` too.

Reply:
Is there any difference between `r'/usr/local'` and `string=True` ?

Reply:
Fixed in https://github.com/spack/spack/pull/18925/commits/b5de5c9d1f5bf2f2cf662ffd32c947e19cbde208

Reply:
Yes, `r'/usr/local'` is identical to `'/usr/local'` since raw strings only prevent parsing of backslashes. `string=True` tells Spack to use string replace instead of re replace.